tests/queries/0_stateless/02932_kill_query_sleep.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+function wait_query_started()
+{
+    local query_id="$1"
+    while [[ $($CLICKHOUSE_CLIENT --query="SELECT count() FROM system.query_log WHERE query_id='$query_id'") == 0 ]]; do
```

Comment:
It's much better to query system.processes. That way you don't need to flush logs.

Reply:
What if the query has failed already?

Then
```
function wait_query_started()
{
    while [[ $($CLICKHOUSE_CLIENT --query="SELECT count() FROM system.processes WHERE query_id='$query_id'") == 0 ]]
```
is going to wait forever.

Reply:
I don't think that's going to be a problem in this test, isn't it?